core/storage/classifier/gae_models_test.py

```diff
@@ -111,6 +111,115 @@ def test_query_new_and_pending_training_jobs(self):
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id06', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id07', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id08', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id09', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id10', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id11', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id12', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id13', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+        classifier_models.ClassifierTrainingJobModel.create(
+            'TextClassifier', 'TextInput', 'exp_id14', 1,
+            next_scheduled_check_time,
+            [{'answer_group_index': 1, 'answers': ['a1', 'a2']}],
+            'state_name2', feconf.TRAINING_JOB_STATUS_NEW, 1)
+
+        training_jobs, offset = (
+            classifier_models.ClassifierTrainingJobModel.
+            query_new_and_pending_training_jobs(offset))
```

Comment:
Suggest just passing in 0 rather than overriding the offset variable. In general, I suggest minimize variable work in tests since it can be more difficult (as a reader) to verify that the test is correct.

Reply:
Done.